packages/abi-gen/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "3.1.1",
+        "changes": [
+            {
+                "note": "Python method parameters are now in snake case; Python wrappers now support tuples in method parameters; document Python method's bytes params as requiring UTF-8; generate Python output into a contract-named folder, not a file (eg exchange/__init__.py rather than exchange.py) leaving space for user-defined additions to the same module, such as for custom types, as used by the Exchange wrapper's manually-wriiten type aliases in the contract_wrappers.exchange.types Python module; support for customizable parameter validation for Python wrappers; wrap Python docstrings better, for pydocstyle compliance; lots of fixes to satisfy linters of generated Python code.",
```

Comment:
```suggestion
                "note": "Python method parameters are now in snake case; Python wrappers now support tuples in method parameters; document Python method's bytes params as requiring UTF-8; generate Python output into a contract-named folder, not a file (eg exchange/__init__.py rather than exchange.py) leaving space for user-defined additions to the same module, such as for custom types, as used by the Exchange wrapper's manually-written type aliases in the contract_wrappers.exchange.types Python module; support for customizable parameter validation for Python wrappers; wrap Python docstrings better, for pydocstyle compliance; lots of fixes to satisfy linters of generated Python code.",
```